libcloud.groovy

```diff
@@ -97,7 +97,7 @@ def replicate_to_clouds(pipecfg, basearch, buildID, stream) {
 def upload_to_clouds(pipecfg, basearch, buildID, stream) {
 
     // Get a list of the artifacts that are currently built.
-    def images_json = readJSON(text: shwrapCapture("cosa meta --get-value images"))
+    def images_json = readJSON(text: shwrapCapture("cosa meta --arch=${basearch} --get-value images"))
```

Comment:
Really, we should also include `--build ${buildID}` in this call since the function is structured as looking generic over any build. And similarly for the replication code. Not new, so not a blocker!

Reply:
added a commit for that